hydra-access-controls/hydra-access-controls.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.4'
 
-  gem.add_dependency 'activesupport', '>= 4', '< 6'
+  gem.add_dependency 'activesupport', '>= 5.2', '< 6.1'
```

Comment:
would this trigger a major release, or...?

Reply:
I was thinking it was a bug since `hydra-head` depends on `rails >= 5.2, <6.1` and we're not testing on rails 4.x anymore, but if someone was depending on `hydra-access-controls` alone then I think it would necessitate a major release.  I'll back this change out then and make it a separate PR.

Reply:
I split this out as #515